typescript/web/src/utils/testing/mock-apollo.tsx

```diff
@@ -0,0 +1,42 @@
+import { PropsWithChildren } from "react";
```

Comment:
We it's still possible, can we rename the `testing` directory as `tests` so that it's consistent with the `__tests__` directories we have everywhere?

Reply:
OK for `stories` 👍